From: Tetsu Yamamoto <tyamamot@redhat.com>
Date: Mon, 4 Aug 2008 15:48:04 -0400
Subject: [xen] ia64: quieter Xen boot
Message-id: 20080804194804.8452.64055.sendpatchset@pq0-1.lab.bos.redhat.com
O-Subject: [RHEL5.3 PATCH 2/10] xen-ia64: Quieter Xen boot
Bugzilla: 430219
RH-Acked-by: Bill Burns <bburns@redhat.com>

bz430219
# HG changeset patch
# User Alex Williamson <alex.williamson@hp.com>
# Date 1180045728 21600
# Node ID 034f3e20ad10cc04935367cf0c8e2d99959b1518
# Parent  c693c50a6ad510157613e3553e49b4191f3e8e76
[IA64] Quieter Xen boot

Demote a couple more printks to debug level output

Signed-off-by: Alex Williamson <alex.williamson@hp.com>

diff --git a/arch/ia64/linux-xen/setup.c b/arch/ia64/linux-xen/setup.c
index 44ad754..e7e9d97 100644
--- a/arch/ia64/linux-xen/setup.c
+++ b/arch/ia64/linux-xen/setup.c
@@ -878,7 +878,7 @@ cpu_init (void)
 	cpu_data = per_cpu_init();
 
 #ifdef XEN
-	printk("cpu_init: current=%p\n", current);
+	printk(XENLOG_DEBUG "cpu_init: current=%p\n", current);
 #endif
 
 	/*
diff --git a/arch/ia64/xen/vhpt.c b/arch/ia64/xen/vhpt.c
index 052570e..d7604e8 100644
--- a/arch/ia64/xen/vhpt.c
+++ b/arch/ia64/xen/vhpt.c
@@ -137,8 +137,8 @@ void vhpt_init(void)
 		panic("vhpt_init: bad VHPT alignment!\n");
 	__get_cpu_var(vhpt_paddr) = paddr;
 	__get_cpu_var(vhpt_pend) = paddr + (1 << VHPT_SIZE_LOG2) - 1;
-	printk("vhpt_init: vhpt paddr=0x%lx, end=0x%lx\n",
-		paddr, __get_cpu_var(vhpt_pend));
+	printk(XENLOG_DEBUG "vhpt_init: vhpt paddr=0x%lx, end=0x%lx\n",
+	       paddr, __get_cpu_var(vhpt_pend));
 	vhpt_erase(paddr);
 	// we don't enable VHPT here.
 	// context_switch() or schedule_tail() does it.
